### ISSUE TITLE
Add txt format (I have reasons)

### ISSUE THREAD

**Issue by nolepguy** [dteviot/WebToEpub#1179]
Please add txt file format as option to export. Of course I can download the epub version and convert it using online converter, but the problem is that the txt file is only one for all chapters (you can easily lose track). It's better for the txt file to be separated per chapters and packed in zip. 

Why txt files? because I can edit it without external software/plugin/whatever it is. Just use notepad.

Why not epub? As I mention earlier, less third-party software is better.

Why not change the .epub to .zip and convert the xhtml to txt? Imagine you have 500 chapters (typically for CN novels) it will be 500 .xhtml files, using online converter you can't convert all the xhtml to txt at once.

Txt makes it easier if you want to translate KR or JP novel. Notepad have built-in search and replace, character counter, line counter, and built-in for every windows user. If you want to translate directly in the raw text, just edit it in txt files without opening multiple software. 

In short: epub is good for readers, txt is good for editor and translator (who likes simplicity)

Thanks for your hard work in making this project. I hope you add this feature.

**Comment by dteviot** [comment]
@nolepguy 

Have you tried Calibre?  Or any of the other options listed here: https://askubuntu.com/questions/102458/how-can-i-convert-epub-files-to-plain-text

Alternate offer, I have another tool https://github.com/dteviot/MergeWebToEpub.  
I can modify that to optionally emit the output as a zip file holding text files.  Note, it is a Windows program.
If acceptable to you, please close this issue and create new one for MergeWebToEpub.  (But please try the other options first.)

Note, I don't want to add this to WebToEpub, because I want to keep WebToEpub as simple as possible.

**Comment by nolepguy** [comment]
@dteviot I have try calibre, and it doesn't work out well for me. I dont know how to convert each chapters in the epub to individual txt file. For other options, I don't know how to install them, it's also the same problem for MergeWebToEPub

**Comment by dteviot** [comment]
@nolepguy 
> it's also the same problem for MergeWebToEPub

You mean you don't know how to install MergeWebToEpub?
Fair enough.  I haven't provided the instructions.  I'll update the readme.

Anyway, If you have a Windows computer, there's a pre-built version you can install with the following steps.
1. Go to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8.
2. Download the MergeWebToEpub zip file
3. Unpack the zip file
4. Double click on MergeWebToEpub.exe

**Comment by nolepguy** [comment]
> @nolepguy
> 
> Have you tried Calibre? Or any of the other options listed here: https://askubuntu.com/questions/102458/how-can-i-convert-epub-files-to-plain-text
> 
> Alternate offer, I have another tool https://github.com/dteviot/MergeWebToEpub. I can modify that to optionally emit the output as a zip file holding text files. Note, it is a Windows program. If acceptable to you, please close this issue and create new one for MergeWebToEpub. (But please try the other options first.)
> 
> Note, I don't want to add this to WebToEpub, because I want to keep WebToEpub as simple as possible.

does the version have this?

**Comment by dteviot** [comment]
@nolepguy 
No I haven't added the ability to export to text files to MergeWebToEpub yet.  I was waiting for your response.

**Comment by nolepguy** [comment]
If you are willing, please kindly add the feature. I'll close this issue.

Pada Kam, 25 Jan 2024 pukul 15.19 dteviot ***@***.***>
menulis:

**Comment by dteviot** [comment]
@nolepguy 
Please go to https://github.com/dteviot/MergeWebToEpub/issues and create an issue.  Then
1. I won't forget it, and
2. You'll be notified when the work is done.
